_component_design/loading-indicators.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Loading indicators
+parent: Processing
+layout: component
+category: Components
+usage: >
+  Loading indicators tell the user that their system is actively retrieving data.
+preview-image: preview-images/loading.svg
+resource: true
+status: Stable
+last-modified: 2017-10-24
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+Use loading indicators when you need provide further visual feedback in cases where a system is making large calls or complex operations in the background.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:""] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-use.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+### Best practice
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- Make sure loading indicators are visible and clearly show the user that information is being retrieved.
+- Center a loading spinner both vertically and horizontally.
+- If loading text is used, always use an ellipses are the word "Loading".
```

Comment:
- Vertically and horizontally center the loading spinner.
"If loading text is used, always use an ellipses are the word "Loading"." - por que?!